test/services/return-requirements/submit-start-date.service.test.js

```diff
@@ -31,13 +31,35 @@ describe('Submit Start Date service', () => {
           licenceHolder: 'Turbo Kid',
           startDate: '2022-04-01T00:00:00.000Z'
         },
-        journey: 'returns-required'
+        journey: 'no-returns-required'
       }
     })
   })
 
   describe('when called', () => {
-    describe('with a valid payload', () => {
+    describe('with a valid payload for currentStartDate', () => {
```

Comment:
Did we mean `currentStartDate`? Or should this be `licenceStartDate`?

Reply:
nice catch, I changed the test but forgot to update that to `licenceStartDate`